core/modules/replica/CreateReplicaJob.h

```diff
@@ -145,7 +145,12 @@ class CreateReplicaJob
      *
      * @see Job::extendedPersistentState()
      */
-    std::list<std::pair<std::string,std::string>> extendedPersistentState() const override;
+    std::list<std::pair<std::string,std::string>> extendedPersistentState() const final;
+
+    /**
+     * @see Job::persistentLogData()
+     */
+    std::list<std::pair<std::string,std::string>> persistentLogData() const final;
```

Comment:
does this also want `override`? (since you reference a base class method in the documentation)